### PR TITLE
fix: should only require CREATE_ADDON when creating addon

### PR DESCRIPTION
**Pull request by ivarconr**
fixes: #2203

### CODE REVIEW

File: frontend/src/component/addons/AddonForm/AddonForm.tsx [hunk 1]
```diff
@@ -334,7 +334,7 @@ export const AddonForm: VFC<IAddonFormProps> = ({
                             type="submit"
                             color="primary"
                             variant="contained"
-                            permission={ADMIN}
+                            permission={editMode ? 'UPDATE_ADDON' : 'CREATE_ADDON'}
```
Comment (nunogois) [inline frontend/src/component/addons/AddonForm/AddonForm.tsx:337]: You can switch the import to grab the `CREATE_ADDON` and `UPDATE_ADDON` constants instead, so we don't have to use string literals: 
image

Comment (ivarconr) [reply frontend/src/component/addons/AddonForm/AddonForm.tsx:337]: will do!

**Review by nunogois — APPROVED** [review]
LGTM

**Review by sjaanus — APPROVED** [review]
LGTM!

**Pull request merged**